pkg/embeddedetcd/config.go

```diff
@@ -92,60 +80,43 @@ func (s *Server) Run(ctx context.Context, peerPort, clientPort string, listenMet
 	cfg.ClientTLSInfo.KeyFile = filepath.Join(cfg.Dir, "secrets", "peer", "key.pem")
 	cfg.ClientTLSInfo.TrustedCAFile = filepath.Join(cfg.Dir, "secrets", "ca", "cert.pem")
 	cfg.ClientTLSInfo.ClientCertAuth = true
-	cfg.ForceNewCluster = forceNewCluster
-	if enableWatchCache {
-		// defines the interval for etcd watch progress notify events.
-		//
-		// note:
-		// - gcp, ocp and upstream k8s set it to 5s, so we simply follow suit
-		// - in practice this value never changes so we are not exposing it as a flag/option
-		// - we enable it only when the watch cache is on otherwise it might not scale
-		//   sending an event every 5s to thousands of clients
-		cfg.ExperimentalWatchProgressNotifyInterval = 5 * time.Second
-	}
+	cfg.ForceNewCluster = o.ForceNewCluster
```

Comment:
what happened to `ExperimentalWatchProgressNotifyInterval` ?

Reply:
do you want to expose it via `EmbeddedEtcd` options?

Reply:
fixed